src/sentry/static/sentry/app/views/settings/project/projectFilters/groupTombstones.jsx

```diff
@@ -23,22 +25,23 @@ class GroupTombstoneRow extends React.Component {
       actor = data.actor;
 
     return (
-      <li className={`group row level-${data.level} type-${data.type}`}>
-        <div className="col-md-10 event-details issue">
+      <PanelItem align="center">
+        <Box flex="1" style={{minWidth: 0}}>
```

Comment:
Should this be a `Content` element instead?

Reply:
@markstory if you're talking about `Content` in `settingsLayout.jsx`, then I don't think so. It's a one-off styled component that does something similar, but I don't think it's worth making it its own component and importing it in.

Reply:
Can we create a local styled component at the very least? I'd like to see less inline styles (unless it's a hassle overriding existing CSS with emotion). We